src/main/resources/assets/paths/daaForDacId.yaml

```diff
@@ -0,0 +1,28 @@
+put:
+  summary: Add DAA to DAC
+  operationId: addDacToDaa
```

Comment:
I would favor removing this. We're not using it for anything since we don't use any code generators that would need it.